Tasks/NodeToolV0/task.json

```diff
@@ -39,6 +39,22 @@
             "defaultValue": "false",
             "required": false,
             "helpMarkDown": "Always checks online for the latest available version that satisfies the version spec. This is typically false unless you have a specific scenario to always get latest. This will cause it to incur download costs when potentially not necessary, especially with the hosted build pool."
+        },
+        {
+            "name": "arch",
+            "type": "pickList",
```

Comment:
Excuse the review, since I'm not a member of Microsoft/Azure team. Just interested in using this myself for a fork of Atom.

I don't see `pickList` in the list of valid data types for a parameter in a template.

Reading from these docs: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#parameter-data-types

If I'm reading correctly, I think the type should just be `string`.

Reply:
Whoops, this is json, not a template YAML file. Please disregard.